OSDOCS-11444: Adds low latency to MicroShift (#80236)
openshift/openshift-docs: modules/microshift-low-latency-kernelrt-conc.adoc

// Module included in the following assemblies:
//
// * microshift_configuring/microshift-low-latency.adoc

:_mod-docs-content-type: CONCEPT
[id="microshift-low-latency-kernelrt-conc_{context}"]

> = Using Red Hat Enterprise Linux for Real-Time

Reviewer: ```suggestion
= Using Red Hat Enterprise Linux for Real Time
```

Reviewer: The "-"is not present here, just in real-time kernel (https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_for_real_time/8/html-single/installing_rhel_8_for_real_time/index)

> If your workload has stringent low-latency determinism requirements for core kernel features such as interrupt handling and process scheduling in the microsecond (μs) range, then {op-system-rt-kernel} is for you. The goal of the {op-system-rtk} is consistent, low-latency determinism that offers predictable response times.

Reviewer 2: >  then {op-system-rt-kernel} is for you. 

Marketing speak. Ick! ;-)
Maybe something like:
```suggestion
If your workload has stringent low-latency determinism requirements for core kernel features such as interrupt handling and process scheduling in the microsecond (μs) range, you can use the {op-system-rt-kernel} for consistent, low-latency determinism that offers predictable response times.
```
